make/mcu/STM32F7.mk

```diff
@@ -164,3 +164,14 @@ MCU_EXCLUDES = \
 
 DSP_LIB := $(ROOT)/lib/main/DSP_Lib
 DEVICE_FLAGS += -DARM_MATH_MATRIX_CHECK -DARM_MATH_ROUNDING -D__FPU_PRESENT=1 -DUNALIGNED_SUPPORT_DISABLE -DARM_MATH_CM7
+
+# MCU dependent optimization
+SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
```

Comment:
Not sure if it makes sense to expect targets to include their own optimisation settings - in my opinion optimisation settings should be managed centrally, and not per target. This will also add another layer of complexity to things like debugging driver timing issues

Reply:
I agree. I'd rather targets don't include there own optimisation settings. Targets are complex enough as it is.